docs/vision-usage.rst

```diff
@@ -214,7 +244,7 @@ in an image.
 
 
 Image Properties
-~~~~~~~~~~~~~~~~
+----------------
```

Comment:
Changes made per the recs? http://documentation-style-guide-sphinx.readthedocs.io/en/latest/style-guide.html#headings